packages/@aws-cdk/cx-api/lib/features.ts

```diff
@@ -104,6 +104,21 @@ export const S3_GRANT_WRITE_WITHOUT_ACL = '@aws-cdk/aws-s3:grantWriteWithoutAcl'
  */
 export const ECS_REMOVE_DEFAULT_DESIRED_COUNT = '@aws-cdk/aws-ecs-patterns:removeDefaultDesiredCount';
 
+/**
+ * ServerlessCluster.clusterIdentifier currently can has uppercase letters,
+ * and ServerlessCluster pass it through to CfnDBCluster.dbClusterIdentifier.
+ * The identifier is saved as lowercase string in AWS and is resolved as original string in CloudFormation.
+ *
+ * If this flag is not set, original value that one set to ServerlessCluster.clusterIdentifier
+ * is passed to CfnDBCluster.dbClusterIdentifier.
+ * If this flag is true, ServerlessCluster.clusterIdentifier is converted into a string containing
+ * only lowercase characters by the `toLowerCase` function and passed to CfnDBCluster.dbClusterIdentifier.
+ *
+ * This feature flag make correct the ServerlessCluster.clusterArn when
+ * clusterIdentifier contains a Upper case letters.
+ */
+export const RDS_CLUSTER_IDENTIFIER_TO_LOWERCASE = '@aws-cdk/aws-rds:clusterIdentifierToLowercase';
```

Comment:
I actually think we might want to name this a little more generically, because I'm pretty sure we'll have the same problem with Instance shortly.

Given that, how about:

```suggestion
export const RDS_LOWERCASE_DB_IDENTIFIER = '@aws-cdk/aws-rds:lowercaseDbIdentifier';
```